edb/schema/delta.py

```diff
@@ -2715,6 +2715,11 @@ class DeleteObject(ObjectCommand[so.Object_T], Generic[so.Object_T]):
     #: in the schema.
     if_unused = struct.Field(bool, default=False)
 
+    #: Potential references to this object that we know are being
+    #: deleted, which we use to resolve if_unused.
+    expiring_refs = struct.Field(AbstractSet[so.Object],
+                                 default=frozenset())
```

Comment:
Should this be an `ephemeral` field?

Reply:
It's not a `schema.Field` on an object, just a delta command field.